modules/mw.cx.MachineTranslationService.js

```
@@ -189,75 +189,69 @@ mw.cx.MachineTranslationService.prototype.cxtoken_error = function () {
-
-	if (this.sourceLanguage === "mdwiki") {
-		var params = {
-			user: mw.user.getName(),
-			wiki: this.targetLanguage,
-			ty: "cxtoken",
-		}
-		const options = {
-			method: 'GET',
-			headers: { 'Accept': 'application/json' }
-		}
+mw.cx.MachineTranslationService.prototype.fetchCXServerToken_MDWIKI_issue = function () {
```

ESLint

[error] Identifier 'fetchCXServerToken_MDWIKI_issue' is not in camel case.

(camelcase)